spec/requests/capitals_controller_spec.rb

```diff
@@ -98,25 +98,25 @@
       context "missing name on bank account" do
         let(:bank_account_params) { attributes_for_list(:liquid_capital_item, 2).map { |account| account.tap { |item| item.delete(:description) } } }
 
-        it_behaves_like "it fails with message", "Missing parameter description"
+        it_behaves_like "it fails with message", "ActiveRecord::RecordInvalid: Validation failed: Description can't be blank"
       end
 
       context "missing lowest balance on bank account" do
         let(:bank_account_params) { attributes_for_list(:liquid_capital_item, 2).map { |account| account.tap { |item| item.delete(:value) } } }
 
-        it_behaves_like "it fails with message", "Missing parameter value"
+        it_behaves_like "it fails with message", "ActiveRecord::RecordInvalid: Validation failed: Value can't be blank"
       end
 
       context "missing description on non_liquid capital" do
         let(:non_liquid_params) { attributes_for_list(:non_liquid_capital_item, 2).map { |nlc| nlc.tap { |item| item.delete(:description) } } }
 
-        it_behaves_like "it fails with message", "Missing parameter description"
+        it_behaves_like "it fails with message", "ActiveRecord::RecordInvalid: Validation failed: Description can't be blank"
```

Comment:
It looks to me like the json-schema validation isn't working as expected if you are getting the ActiveRecord::RecordInvalid error. (To me this implies we are either not calling json_validator before trying to save the record or the validation is not correct).